azureLogIngestion/opentelemetry/index.ts

```diff
@@ -162,7 +222,7 @@ export default class OpenTelemetryAdapter {
         span.end(endTimeHrFromDuration(appSpan.timestamp, appSpan.durationMs))
         // OT batch processor doesn't give access to current batch size
         // or batch content. This lets us do snapshot tests.
-        this.currentBatch.push(span)
+        this.currentBatch.push(loggableSpan(span))
```

Comment:
Is there a way to detect we're in a test suite and only do this there? Not a big deal for now.

Reply:
@kolanos Arguably we could set an env var in one of the tests and base this off of that. 

I do like the idea of pushing something into the currentBatch, though, as a way of detecting whether or not there are otel spans to flush.